CryptoTechReminderSystem.Main/Program.cs

```diff
@@ -81,6 +83,7 @@ private void ScheduleJobs()
         {
             JobManager.AddJob(RemindLateBillablePeopleJob, s => s.ToRunOnceAt(10, 0).AndEvery(30).Minutes());
             JobManager.AddJob(ListLateBillablePeopleJob, s => s.ToRunOnceAt(12, 30));
+            JobManager.AddJob(RemindProjectManagersJob, s => s.ToRunOnceAt(13, 00));
```

Comment:
1300 chosen on advice from Chris Blackburn.  Can be updated.

Reply:
If you're able to get the reasoning of why 13:00, might be useful to document as a comment.

Reply:
That's a really good point Adrian.  I didn't get the 1pm reasoning from Chris at the time.  When this goes 'live' and is actually reminding DMs I'll post something in #common-room asking for feedback on the time the message should go out. 

I've also added a TODO to remind me 👍